Revert 520c0f867e06de5bd03a90b535e5665b385c5a36 since bindgen 0.59 is already
packaged in Fedora, and it was only updated to the 0.60 branch to fix tests
which can't run due to other failures.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,7 @@
 version = "^1.0.0"
 
 [build-dependencies.bindgen]
-version = "^0.60.1"
+version = "^0.59.1"
 features = ["runtime"]
 default-features = false
 
